installation/resources/installer-cr-cluster.yaml.tpl

```diff
@@ -23,8 +23,14 @@ spec:
       namespace: "kyma-installer"
     - name: "istio-kyma-patch"
       namespace: "istio-system"
+    - name: "knative-serving-crds-init"
```

Comment:
I don't like to have a dedicated module jsut for the migration logic, please double-check how others are doing it and try to find a better place.

Reply:
done.